bookshop/src/store/index.js

```diff
@@ -3,26 +3,30 @@ import Vuex from 'vuex'
 import axios from 'axios'
 import _ from 'lodash'
 import Cart from '../dao/Cart'
+import config from 'Config'
 
 Vue.use(Vuex)
 
 const store = new Vuex.Store({
   state: {
     books: [],
+    book: null,
     isLoggedIn: false,
     genres: [],
     authors: [],
     mcart: new Cart(),
-    user: null
+    user: null,
+    spinner: false
   },
   mutations: {
-    set (state, {type, items}) {
-      state[type] = items
+    set (state, {type, data}) {
+      state[type] = data
     },
     setCart (state, items) {
       state.mcart.items = items
     },
-    addToCart (state, item) {
+    addToCart (state) {
```

Comment:
I think the best way is to look another an this issue.
If we use global store in our project then it should contain all selected books for adding to card.
So it can be array of books and 'addToCart ' manages it. 
Now it reload card. It should be moved to Card component`s methods.

Reply:
Not sure that we need selectedBook as global state. But ok.

Reply:
Cool